client/src/main/java/org/apache/rocketmq/client/ClientConfig.java

```diff
@@ -173,12 +177,20 @@ public void setVipChannelEnabled(final boolean vipChannelEnabled) {
         this.vipChannelEnabled = vipChannelEnabled;
     }
 
+    public boolean isUseTLS() {
+        return useTLS;
+    }
+
+    public void setUseTLS(boolean useTLS) {
+        this.useTLS = useTLS;
+    }
+
     @Override
     public String toString() {
         return "ClientConfig [namesrvAddr=" + namesrvAddr + ", clientIP=" + clientIP + ", instanceName=" + instanceName
             + ", clientCallbackExecutorThreads=" + clientCallbackExecutorThreads + ", pollNameServerInterval=" + pollNameServerInterval
             + ", heartbeatBrokerInterval=" + heartbeatBrokerInterval + ", persistConsumerOffsetInterval="
             + persistConsumerOffsetInterval + ", unitMode=" + unitMode + ", unitName=" + unitName + ", vipChannelEnabled="
-            + vipChannelEnabled + "]";
+            + vipChannelEnabled + ", useTLS=" + useTLS + "]";
```

Comment:
I do not get your point. You mean change "+" to a chain of append() method invocations?